exampleSite/content/shortcodes/includes.md

```diff
@@ -0,0 +1,51 @@
+Include Shorcode can include files of different types. By specifying a language, the included file will have syntax highligting.
+
+## Shortcode
+
+```tpl
+{{</* include file="relative/path/from/hugo/root" language="go" markdown=[false|true] */>}}
+```
+
+Attributes:
+
+| Name | Usage | default |
+|---|---|---|
+|  file | path to the included file relative to the hugo root | empty value |
+| language* | language for [syntax highlighting](https://gohugo.io/content-management/syntax-highlighting/#list-of-chroma-highlighting-languages)  | empty value |
+| markdown | included file is markdown | false |
+| options | highlighting [options](https://gohugo.io/content-management/syntax-highlighting/#highlight-shortcode) | linenos=false |
+
+* if not set, the content will be rendered as plain html
```

Comment:
Should be escaped, is rendered as bullet point otherwise

Reply:
ok, done